### PR TITLE
Created all classes same as in the video and added own classes User a…

### PR DESCRIPTION
**Pull request by YuraMitriakhin**
…nd UserDao

### CODE REVIEW

**Review by Mandybur800 — APPROVED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: What's missing?

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            if (command.equalsIgnoreCase("q")) {
+                break;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException e) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: What else exception you may have?

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+                System.out.println("Please input correct data!");
+            }
+            betDao.add(bet);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: let's not add bet if it's null

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 1 above)
Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: and what's about UserDao?

File: src/main/java/core/basesyntax/model/User.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.model;
+
+import java.util.List;
+
+public class User {
+    private int id;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/model/User.java]: in the Future all of our entities will have id's, but now let's remove it

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/model/User.java (hunk 4 above)
Comment (YuraMitriakhin) [reply src/main/java/core/basesyntax/model/User.java]: Ok, it's just a habit)

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 1 above)
Comment (YuraMitriakhin) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: fixed

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 2 above)
Comment (YuraMitriakhin) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: fixed

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 3 above)
Comment (YuraMitriakhin) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: ok, now I add bet to user

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**